examples/image_classifier/resnet_18/README.md

```diff
@@ -23,7 +23,7 @@ Ex:  `cd  examples/image_classifier/resnet_18`
 In this example , we use the following config
 
 ```
-echo "pt2 : {backend: inductor, mode: reduce-overhead}" > model-config.yaml
+echo "pt2:\n  compile:\n    enable: True\n    backend: inductor\n    mode: reduce-overhead" > model-config.yaml
```

Comment:
If you just use multiple lines people will be able to visualize the new format much better:
```bash
echo "pt2:
  compile:
    enable: True
    backend: inductor
    mode: reduce-overhead" > model-config.yaml
```